examples/pytorch/graphsage/train_sampling.py

```diff
@@ -214,6 +225,8 @@ def run(args, device, data):
 
     if args.dataset == 'reddit':
         g, n_classes = load_reddit()
+    elif args.dataset == 'ogb-products':
```

Comment:
can we use its original name 'ogbn-products'?

Reply:
Changed to 'ogbn-products' in fc37138